theano/tensor/basic.py

```diff
@@ -1266,28 +1266,19 @@ class MaxAndArgmax(Op):
 
     def make_node(self, x, axis=None):
         x = _as_tensor_variable(x)
-
-        if isinstance(axis, (tuple, list)):
+        if isinstance(axis, (tuple, list, numpy.ndarray)):
+            # List of axes: make them non-negative, and sort them
             axis = [int(a) for a in axis]
-            if len(axis) != 1:
```

Comment:
Why did you remove the sorting and move to positiv axis? I think we should keep it if possible.